File: algorithms/268.cpp
#include <iostream>
#include <vector>
using namespace std;

class Solution
{
public:
    int missingNumber(vector<int>& nums)
    {
        int n = nums.size();
        int k = n*(n+1)/2;
        for ( int i = 0; i < n; ++i )
            k -= nums[i];
        return k;
    }
};

int main()
{
    return 0;
}

File: algorithms/python/268.py
class Solution(object):
    def missingNumber(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        n = len(nums)
        s = n * (n+1) // 2
        for i, n in enumerate(nums):
            s -= n
        return s
